File: 250803/상품 코드/product-code.java
import java.util.*;

public class Main {

    static class Product {
        
        String name;
        int price;

        public Product(){
            this.name = "codetree";
            this.price = 50;
        }

        public Product(String name, int price){
            this.name = name;
            this.price = price;
        }
    }

    public static void main(String[] args) {
        // Please write your code here.
        Scanner sc = new Scanner(System.in);

        String name = sc.next();
        int price = sc.nextInt();

        Product p1 = new Product();
        Product p2 = new Product(name, price);

        System.out.println("product " + p1.price + " is " + p1.name);
        System.out.println("product " + p2.price + " is " + p2.name);


    }
}

File: 250803/상품 코드/product-code.py
class Codetree :
    def __init__ (self, name = "codetree", price = 50) :
        self.name = name;
        self.price = price;


name, price = map(str, input().split());

price = int(price);

codetree1 = Codetree();
codetree2 = Codetree(name, price);

print(f"product {codetree1.price} is {codetree1.name}")
print(f"product {codetree2.price} is {codetree2.name}")

